lib/email/cancel.ts

```diff
@@ -34,8 +35,9 @@ export async function sendCancelationEmail(booking: Booking) {
 
     return smtp.sendMail({
         to: booking.email,
+        replyTo: Config.REPLY_TO,
```

Comment:
Da dieser Header überall genutzt werden soll, wäre das global in `lib/smtp.ts` besser.